config/localTemplate.go

```diff
@@ -76,7 +77,58 @@ type Local struct {
 	BaseLoggerDebugLevel uint32 `version[0]:"1" version[1]:"4"`
 	// if this is 0, do not produce agreement.cadaver
 	CadaverSizeTarget uint64 `version[0]:"1073741824" version[24]:"0"`
-	CadaverDirectory  string `version[27]:""`
+	// if this is not set, MakeService will attempt to use ColdDataDir instead
+	CadaverDirectory string `version[27]:""`
+
+	// Optional Data Directories
+	// data dirs are used to house data related to the operation of the node.
+	// by default, the only data directory that is required is the datadir,
+	// specified by -d or the os environment variable algorand_data.
+	// by specifying additional data directories, the node will store data in those directories,
+	// *instead* of the datadir. This is useful for example, if you want to store the ledger
+	// on a fast SSD, but store the logs on a slower spinning disk.
+	// In order to allow for nodes to support multiple networks, these data directories are not used directly,
+	// but instead will hold subdirectories for each network, named by its genesis id.
+	// once these paths are resolved to absolute paths with genesis directories, they are passed to the node to use.
+	// For simple node operation: do not specify any of these directories.
+	// For advanced node operation: specify the HotDataDir and ColdDataDir to separate drives
+	// For expert node operation: specify any of the directories to customize the node's data storage.
+
+	// Hot Data Directory stores data that is frequently accessed by the node. This includes:
+	// - ledger
+	// - log file
+	// if not specified, the node will use the datadir for these files.
+	// these individual resources may themselves have their own directories specified, which would override this
+	HotDataDir string `version[29]:""`
+
+	// Cold Data Directory stores data that is infrequently accessed by the node. This includes
+	// - block database
+	// - catchpoint database
+	// - stateproof database
+	// - crash database
+	// - tracker database
+	// - log archive
+	// - participation db
+	// if not specified, the node will use the datadir for these files
+	// these individual resources may themselves have their own directories specified, which would override this
+	ColdDataDir string `version[29]:""`
+
+	// TrackerDbDir stores the tracker database. if not specified, the node will use the hotdatadir
```

Comment:
Keep naming consistent with what defined above - here and below
```suggestion
	// TrackerDbDir stores the tracker database. if not specified, the node will use the HotDataDir
```